tripal/config/schema/tripal.schema.yml

```diff
@@ -100,3 +100,161 @@ field.field.tripal_entity.*.*:
         termAccession:
           type: string
           label: 'Term Accession'
+
+tripal.tripal_content_types.*:  
```

Comment:
I just realized I accidentally committed this.  I think it's okay to leave in and doesn't need review because it won't break anything, isn't used yet, and I'll be adding a new  PR to support this soon.

Reply:
We should likely remove this from this PR because I've found that Drupal is very aggressive checking these schema files and causing testing to fail --SOMETIMES EVEN IF the schema file is not being used yet.